go/private/BUILD.sdk.bazel

```diff
@@ -118,6 +118,7 @@ exports_files(
     glob([
         "lib/time/zoneinfo.zip",
         "misc/wasm/**",
+        "lib/wasm/**",
```

Comment:
Could you extract the two `wasm` glob patterns into a separate glob call that allows empty matches?